components/Object/Card/Visual/ObjectCardVisualDefault.vue

```diff
@@ -1,19 +1,80 @@
 <script setup lang="ts">
 import type { BoardCard } from '~/types/Board'
 
-defineProps<{
+const props = defineProps<{
   boardCard: BoardCard
 }>()
+
+const currentHealth = ref(props.boardCard.card.health)
+const maxWidth = 70 - 16
+
+const text = ref<HTMLElement | null>(null)
```

Comment:
Mittlerweile ist die Empfehlung von VUe `useTemplateRef` zu nutzen:

```suggestion
const text = useTemplateRef<HTMLElement>('text')
```